tools/correlation_analysis/abims_correlation_analysis.xml

```diff
@@ -0,0 +1,322 @@
+<tool id="correlation_analysis" name="Metabolites Correlation Analysis" version="1.0.1+galaxy0" >
+
+    <description></description>
```

Comment:
describtion missing?

Reply:
@melpetera Help here, please

Reply:
Done! Not so easy to be short and informative.

Reply:
Cool, thanks https://github.com/workflow4metabolomics/tools-metabolomics/pull/189/commits/e532ba7ae2bda29738991a34ab41d15fe50f9161